securemeet.py

```
@@ -0,0 +1,986 @@
+        if not self.is_recording: return
+        ts = time.time() - self.start_time
+        self.transcript_text.config(state=tk.NORMAL)
+        self.transcript_text.insert(tk.END, f"\n--- Bookmark @ {ts:.1f}s ---\n")
+        self.transcript_text.config(state=tk.DISABLED)
+
+    # Audio processing loop -------------------------------------------------------
+    def process_audio_loop(self):
+        while not self.stop_processing.is_set():
+            data = self.audio_processor.get_audio_data()
+            if data is None:
+                time.sleep(0.05); continue
```

Ruff (0.15.11)

[error] Multiple statements on one line (semicolon)

(E702)